### PR TITLE
Updated Types for Parsers and Parser Utils

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

This PR updates the types for Parsers and its utility functions

### CODE REVIEW

**Comment by adithyaakrishna** [comment]
Parser Utils is giving a type error, will fix it first thing tomorrow morning :)

**Comment by adithyaakrishna** [comment]
@dkoes Tests Passed 🎉

File: src/parsers/PQR.ts [hunk 1]
```diff
@@ -1,17 +1,8 @@
+import { ParserOptionsSpec } from "./ParserOptionsSpec";
 import { assignPDBBonds } from "./utils/assignPDBBonds";
 import { computeSecondaryStructure } from "./utils/computeSecondaryStructure";
 
-    /**
```
Comment (dkoes) [inline src/parsers/PQR.ts:4]: why are you removing documentation?

File: src/parsers/PRMTOP.ts [hunk 2]
```diff
@@ -1,16 +1,6 @@
 // @ts-nocheck
-/**
```
Comment (dkoes) [inline src/parsers/PRMTOP.ts:2]: do not remove documentation, the applies across the commit

**Review by dkoes — CHANGES_REQUESTED** [review]
Put documentation back in.

**Comment by adithyaakrishna** [comment]
@dkoes Thanks for pointing them out, I had missed reverting those changes back. I have updated the docs with the above commit

**Review by dkoes — APPROVED** [review]

**Pull request merged**